### PR TITLE
CXTB-417 automate senior app deactivated account status visuals

### PR DESCRIPTION
**Pull request by csanchez-nts**
- added test
proof of succeed:
image

### CODE REVIEW

File: tests/cases/seniors_app/case_seniors_app_account_statuses.yaml [hunk 1]
```diff
@@ -455,3 +455,61 @@ TestSeniorAppActiveAccountStatusAndBehavior:
         - Type: ne
           Var: SECOND_ERROR_MESSAGE_VAR
           Value: "An unknown server-side error occurred while processing the command. Original error: Unable To Rotate Device"
+
+# CXTB-417 Senior app - Verify “Deactivated” Account Status Visuals
+TestSeniorAppDeactivatedAccountStatusAndBehavior:
```
Comment (csanchez-nts) [inline tests/cases/seniors_app/case_seniors_app_account_statuses.yaml:460]: Name should be according to test case and scenario title. TestSeniorAppDeactivatedAccountStatusVisuals

File: tests/cases/seniors_app/case_seniors_app_account_statuses.yaml
Comment (csanchez-nts) [inline tests/cases/seniors_app/case_seniors_app_account_statuses.yaml:1]: Add test to set_non_physical_mobile_tests set file

File: tests/cases/seniors_app/case_seniors_app_account_statuses.yaml (hunk 1 above)
Comment (jmontero-nts) [reply tests/cases/seniors_app/case_seniors_app_account_statuses.yaml]: solved in this comit https://github.com/neveralone-chs-org/TestRay/pull/197/commits/8ee0c37915c677a2c5c31aa42ddbe282094580c2

File: tests/cases/seniors_app/case_seniors_app_account_statuses.yaml
Comment (jmontero-nts) [reply tests/cases/seniors_app/case_seniors_app_account_statuses.yaml:1]: solved in this commit https://github.com/neveralone-chs-org/TestRay/pull/197/commits/8d663fa474a5bd7f0dacab37c218c03985db9951

File: tests/cases/seniors_app/case_seniors_app_account_statuses.yaml [hunk 2]
```diff
@@ -455,3 +455,61 @@ TestSeniorAppActiveAccountStatusAndBehavior:
...
+      App: desktop    
+  Actions:
+  # Calling Onboarder Login case
+    - Type: case
+      Value: OnboarderCarePartnerLogin
+  # Calling Senior Login case
+    - Type: case
+      Value: LoginNeverAloneHelper
+  # Change Onboarder to Deactivated
+    - Type: set_env_var
```
Comment (csanchez-nts) [inline tests/cases/seniors_app/case_seniors_app_account_statuses.yaml:484]: See first review comment

File: tests/cases/seniors_app/case_seniors_app_account_statuses.yaml [hunk 3]
```diff
@@ -455,3 +455,61 @@ TestSeniorAppActiveAccountStatusAndBehavior:
         - Type: ne
           Var: SECOND_ERROR_MESSAGE_VAR
           Value: "An unknown server-side error occurred while processing the command. Original error: Unable To Rotate Device"
+
+# CXTB-417 Senior app - Verify “Deactivated” Account Status Visuals
+TestSeniorAppDeactivatedAccountStatusVisuals:
+  Environment: Settings
+  Vars:
+    LOGOUT_OPTION: "Go Off Shift"
+    ACCOUNT_STATUS: 'active' #this is for the precase
```
[the remark this answers is not in the document]
Comment (jmontero-nts) [reply tests/cases/seniors_app/case_seniors_app_account_statuses.yaml:464]: because the user would be desactivated at the first try

**Review by jmontero-nts — APPROVED** [review]

**Pull request merged**